components/tidb_query/src/rpn_expr/impl_cast.rs

```diff
@@ -21,98 +24,116 @@ fn get_cast_fn_rpn_meta(
     let from = box_try!(EvalType::try_from(from_field_type.as_accessor().tp()));
     let to = box_try!(EvalType::try_from(to_field_type.as_accessor().tp()));
     let func_meta = match (from, to) {
+        // ---------------------* as real-----------------------
```

Comment:
You can use a blank line to separate them if necessary (`---------------------* as real-----------------------` seems ugly).